samples/unity-of-bugs/Assets/Scripts/SentryOptionConfiguration.cs

```diff
@@ -0,0 +1,15 @@
+using Sentry;
+using Sentry.Unity;
+
+public class SentryOptionConfiguration : SentryOptionsConfiguration
+{
+    public override void Configure(SentryUnityOptions options)
+    {
+        // Here you can programmatically modify the Sentry option properties used for the SDK's initialization
+
+        options.Environment = "dev";
+
+        options.NativeOverrides.Environment = "native_dev";
```

Comment:
Is this a real use case though?
Shouldn't we just take the `dev` above and automatically override native?